guides/source/action_cable_overview.md

```diff
@@ -784,7 +784,7 @@ The Redis adapter also supports SSL/TLS connections. The required SSL/TLS parame
 ```
 production:
   adapter: redis
-  url: rediss://10.10.3.153:tls_port
+  url: redis://10.10.3.153:tls_port
```

Comment:
This is correct as-is. `rediss` is the [Redis-over-TLS scheme name](https://www.iana.org/assignments/uri-schemes/prov/rediss).